src/__tests__/attributes-shorthand.input.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+// To prevent warnings in console from react
+const Custom = () => null
```

Comment:
You could also render an `input(checked)` so we can actually see html output in the test.

Reply:
@ForbesLindesay I started using `data-` attributes to keep html output. Let me know if you think it does not appropriate.